src/app/COMPONENTS/user-details/user-details.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component } from '@angular/core'
+import { MaterialModule } from '../../material.module'
+import { UsersService } from '../../service/users/users.service'
+import { ActivatedRoute } from '@angular/router'
+import { BehaviorSubject, map, Observable, switchMap, take, tap } from 'rxjs'
+import { User, UpdateUserRoles } from '../../models/user.model'
+import { CommonModule } from '@angular/common'
+import { FormBuilder, FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms'
+
+@Component({
+  selector: 'app-user-details',
+  standalone: true,
+  imports: [
+    MaterialModule,
+    CommonModule,
+    ReactiveFormsModule
+  ],
+  templateUrl: './user-details.component.html',
+  styleUrl: './user-details.component.scss'
+})
+export class UserDetailsComponent {
+
+  individualUserSubject: BehaviorSubject<User | null> = new BehaviorSubject<User | null>(null)
+  individualUser$: Observable<User | null> = this.individualUserSubject.asObservable()// this should be the value whitch we put in async pipe in details template
```

Comment:
you can just directly use the getIndividualUser() method for this observable and use it in the template